vocab/index.html

```diff
@@ -160,42 +230,111 @@ <h3>abc</h3>
 					</tr>
 					<tr>
 						<th>Cardinality:</th>
-						<td>One or more</td>
+						<td>Zero or more</td>
+					</tr>
+					<tr>
+						<th>Example:</th>
+						<td>
+							<pre>&lt;meta property="brl:producer">APH&lt;/meta></pre>
+						</td>
+					</tr>
+				</table>
+			</section>
+
+			<section id="volumes">
+				<h3>volumes</h3>
+
+				<table class="tabledef">
+					<caption>Definition of the <code>volumes</code> property</caption>
+					<tr>
+						<th>Name:</th>
+						<td>
+							<code>volumes</code>
+						</td>
+					</tr>
+					<tr>
+						<th>Description:</th>
+						<td>Identifies the total number of volumes within the eBraille publication. The number typically
+							corresponds to the number of XHTML content documents in the publication.</td>
+					</tr>
+					<tr>
+						<th>Allowed value(s):</th>
+						<td>
+							<code>xsd:unsignedInt</code>
+						</td>
+					</tr>
+					<tr>
+						<th>Cardinality:</th>
+						<td>Zero or one</td>
 					</tr>
 					<tr>
 						<th>Example:</th>
 						<td>
-							<pre>&lt;meta
-    property="brl:abc">
-   ...
-&lt;/meta></pre>
+							<pre>&lt;meta property="brl:volumes">3&lt;/meta></pre>
 						</td>
 					</tr>
 				</table>
 			</section>
 		</section>
-		<section id="vocab-rendering">
-			<h2>Rendering properties</h2>
+		<section id="vocab-content">
+			<h2>Content properties</h2>
 
-			<section id="xyz">
-				<h3>xyz</h3>
+			<section id="dotNumber">
+				<h3>dotNumber</h3>
 
 				<table class="tabledef">
-					<caption>Definition of the <code>xyz</code> property</caption>
+					<caption>Definition of the <code>dotNumber</code> property</caption>
 					<tr>
 						<th>Name:</th>
 						<td>
-							<code>xyz</code>
+							<code>dotNumber</code>
 						</td>
 					</tr>
 					<tr>
 						<th>Description:</th>
-						<td>Identifies ...</td>
+						<td>Identifies whether 6 or 8 dot braille cell characters are used in the text.</td>
 					</tr>
 					<tr>
 						<th>Allowed value(s):</th>
 						<td>
-							<code>xsd:string</code>
+							<code>6</code> | <code>8</code>
+						</td>
+					</tr>
+					<tr>
+						<th>Cardinality:</th>
+						<td>Exactly one</td>
+					</tr>
+					<tr>
+						<th>Example:</th>
+						<td>
+							<pre>&lt;meta property="brl:dotNumber">6&lt;/meta></pre>
+						</td>
+					</tr>
+				</table>
+			</section>
+
+			<section id="graphicType">
+				<h3>graphicType</h3>
+
+				<table class="tabledef">
+					<caption>Definition of the <code>graphicType</code> property</caption>
+					<tr>
+						<th>Name:</th>
+						<td>
+							<code>graphicType</code>
+						</td>
+					</tr>
+					<tr>
+						<th>Description:</th>
+						<td>
+							<p>Identifies the format(s) of any tactile graphics included in the work.</p>
+							<p>Repeat the property for each different format.</p>
+						</td>
+					</tr>
+					<tr>
+						<th>Allowed value(s):</th>
+						<td>
+							<code>JPG</code> | <code>PDF</code> | <code>PNG</code> | <code>SVG</code>
 						</td>
```

Comment:
Wouldn't we will want Cardinality here of zero or one?

Reply:
Since you can have tactical graphics in multiple formats, you need to allow multiple instances of the property.

EPUB doesn't define a way of parsing metadata properties, so if you try to overload all the values in one string, then you open the can of worms that is parsing that back into tokens. The typical approach is to repeat a property once for each unique value (like identifiers, language, creators, access modes, etc.).

Reply:
Right, but don't we want to say that you can have 0 or more of these separate metadata properties, each of which would have one value for the type of tactile graphic?

Reply:
Sorry, I read your first comment as asking for zero or one.

I was going by the word doc @wfree-aph sent around which has this as a required field. That means there must be at least one. Did that change in the discussions yesterday?

Reply:
I would think that there may not be any because there may not be a tactile image of any kind in the eBRF file. @wfree-aph don't you agree?  Is the original document Matt is referencing in error?

Reply:
No, this is my error. It's in the required section of the document but is only conditionally required when tactileGraphic is set to true. My reading skills are a bit diminished this week...☹️

I've fixed the definition.

Reply:
Sorry for my lateness on this discussion but glad it was worked out.